lib/phobos.rb

```diff
@@ -60,16 +60,22 @@ def configure_logger
       log_file = config.logger.file
       ruby_kafka = config.logger.ruby_kafka
       date_pattern = '%Y-%m-%dT%H:%M:%S:%L%zZ'
-      file_layout = Logging.layouts.json(date_pattern: date_pattern)
-      stdout_layout = Logging.layouts.pattern(date_pattern: date_pattern)
+      json_layout = Logging.layouts.json(date_pattern: date_pattern)
+
+      stdout_layout = if config.logger.stdout_format&.to_sym == :json
+        json_layout
+      else
+        Logging.layouts.pattern(date_pattern: date_pattern)
+      end
+
       appenders = [Logging.appenders.stdout(layout: stdout_layout)]
 
       Logging.backtrace(true)
       Logging.logger.root.level = silence_log ? :fatal : config.logger.level
 
       if log_file
         FileUtils.mkdir_p(File.dirname(log_file))
-        appenders << Logging.appenders.file(log_file, layout: file_layout)
+        appenders << Logging.appenders.file(log_file, layout: json_layout)
```

Comment:
This will change log-format in files even if you don't configure stdout format to json. The PR description doesn't mention changing the file log format.

Reply:
This is just a name change to the variable. It used to be called file_layout but since the ACTUAL layout is always JSON, we think a better name is json_layout. 

In case of the name choice `stdout_layout`, this is not a fixed layout but either one of two possibilities. Hence we just called it stdout_layout to not lull anyone into thinking it is fixed.

Reply:
I see now.